wsd/ClientSession.hpp

```diff
@@ -70,6 +72,11 @@ class ClientSession final : public Session
     /// Handle kit-to-client message.
     bool handleKitToClientMessage(const std::shared_ptr<Message>& payload);
 
+    std::optional<bool>
+    handleDocNotPassProtected(const std::shared_ptr<Message>& payload,
```

Comment:
"Pass" can be ambiguous. Better to use "Password". Though an even better name would not reference passwords at all, since this has nothing to do with that. Perhaps a better name is `handleKitToClientMessageOnOpenDoc()` or shorter `handleOpenDocKitToClientMessage()` ?

The idea is that the function is handling Kit-to-Client messages on Open Documents.